plugins/woocommerce/client/legacy/js/jquery-tiptip/jquery.tipTip.js

```diff
@@ -98,8 +98,12 @@
 				}
 
 				function active_tiptip(){
+					var content = typeof opts.content === 'function' ? opts.content() : org_title;
+					if (!content) {
+						return;
+					}
 					opts.enter.call(this);
-					tiptip_content.html(org_title);
+					tiptip_content.html(content);
```

Comment:
For the sake of documenting the changes in this file, maybe we can create a P2 noting that we are changing our instance of `tipTip`. The [version we're using](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/client/legacy/js/jquery-tiptip/jquery.tipTip.js#L7) is pretty old, and I think updating it probably won't be in our plans.

Reply:
I would also put a comment at the top of this file clearly mentioning that this is a custom version of the jQuery plugin. The header still would lead people to believe that is it the stock 1.3 version.

Reply:
Hey @octaedro, it's old, but it's actually the latest version. The plug-in is not updated anymore.

@mattsherman Regarding the comment at the top, I agree. I'll add the comment.